### ISSUE TITLE
Wappalyzer is intermittently causing websites to load slowly/partially/not at all

### ISSUE THREAD

**Issue by thelibrariancat** [AliasIO/Wappalyzer#2398]
**Do you want to request a *feature*, a *new application detection* or report a *bug*?**
Bug

**Is your issue about WebExtension driver (Chrome & Firefox), the website, the NPM driver or the bookmarklet ?**
WebExtension - Firefox

**What is the current behavior ?**
When Wappalyzer is enabled, websites load very slowly, if at all. When I disable the extension, the problem goes away.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a demo of the problem.**
Please note this is an intermittent problem; it's hard to reproduce reliably. Open Firefox, try going to a website. The site will either load very slowly; partially and then stop; or not at all. ("Slowly" in this case means a minute or more.)

**What is the expected behavior ?**
Websites should load at a decent speed whether or not I have Wappalyzer enabled.

**Which versions of Wappalyzer, and which browser / OS are affected by this issue ? Did this work in previous versions ?**
I first noticed this with version 5.5.2 (July 15), on Firefox/Windows 7. I did not have trouble with previous versions.

**Comment by KakersUK** [comment]
I've been having this problem for the last few weeks as well. Thank god I've found what the problem is. I normally have quite a few tabs open so I've been able to re-create this daily.

Browsing sites suddenly all my tabs no longer load websites as if I've lost networking but other browsers without extensions work fine. As soon as I disable this plugin in Firefox everything comes back. I've even tried turning it on again and re-installing as soon as I do the problem comes back.

I've now have to completely remove this plugin which is a shame as I use it all the time! :(
